internal/hud/hud.go

```diff
@@ -196,6 +196,9 @@ func (h *Hud) handleScreenEvent(ctx context.Context, dispatch func(action store.
 			url := h.webURL
 			url.Path = fmt.Sprintf("/r/%s/", r.Name)
 			_ = browser.OpenURL(url.String())
+			dispatch(OpenLogAction{
```

Comment:
there's no good reason to put this in EngineState, create actions for it, etc. It's a lot of indirection that buys you nothing. just inject analytics into the HUD and call Incr() directly.